Estore.UITests/Features/CreateUsersTests.feature

```diff
@@ -0,0 +1,136 @@
+	And User page is open
+	And Admin click on the Add User button
+	When Create user modal window is open
+	And Admin click on the Cancel button
+	Then Create user modal window should be close
+
+#US_49_26
+@AdminLoggedIn
+Scenario: US_49_26_Admin adds a new customer
+	Given Admin click on the Users button
+	And User page is open
+	And Admin click on the Add User button
+	When Create user modal window is open
+	And Admin fills modal window and registers new customer
+	Then Create user modal window should be close
+	Then New customer appeared in the users list
+	And Info message 'User was added' is presented
+
+#US_49_27
+@AdminLoggedIn
+Scenario: US_49_27_Admin leaves First name field empty
+	Given Admin click on the Users button
+	And User page is open
+	And Admin click on the Add User button
+	When Create user modal window is open
+	And Admin fills 'First name' input field: '' 
+	Then Help message under 'First name' field should be 'First name is required!'
+
+#US_49_28
+@AdminLoggedIn
+Scenario: US_49_28_Admin leaves Last name field empty
+	Given Admin click on the Users button
+	And User page is open
+	And Admin click on the Add User button
+	When Create user modal window is open
+	And Admin fills 'Last name' input field: '' 
+	Then Help message under 'Last name' field should be 'Last name is required!'
+	
+#US_49_29
+@AdminLoggedIn
+Scenario: US_49_29_Admin leaves Email field empty
+	Given Admin click on the Users button
+	And User page is open
+	And Admin click on the Add User button
+	When Create user modal window is open
+	And Admin fills 'Email' input field: '' 
+	Then Help message under 'Email' field should be 'Email is required!'
+
+#US_49_30
+@AdminLoggedIn
+Scenario: US_49_30_Admin leaves Password field empty
+	Given Admin click on the Users button
+	And User page is open
+	And Admin click on the Add User button
+	When Create user modal window is open
+	And Admin fills 'Password' input field: '' 
+	Then Help message under 'Password' field should be 'Password is required!'
+
+#US_49_31a
+@AdminLoggedIn
+Scenario: US_49_31a_Admin leaves Repeat password field empty
+	Given Admin click on the Users button
+	And User page is open
+	And Admin click on the Add User button
+	When Create user modal window is open
+	And Admin fills 'Repeat password' input field: '' 
+	Then Help message under 'Repeat password' field should be 'Repeat password is required!'
+	
+#US_49_31b
+@AdminLoggedIn
+Scenario: US_49_31b_Admin fills Password and Repeat password fields different values
+	Given Admin click on the Users button
+	And User page is open
+	And Admin click on the Add User button
+	When Create user modal window is open
+	And Admin fills 'Password' input field: 'randomValue'
+	And Admin fills 'Repeat password' input field: 'randomValue' 
+	Then Help message under 'Repeat password' field should be 'Passwords don"t match'
+		
+#US_49_32
+@AdminLoggedIn
+Scenario: US_49_32_Admin fills Password field with invalid value
```

Comment:
it would be better to use scenario outline

Reply:
fixed